core/test/time.jl

```diff
@@ -45,7 +45,7 @@ end
 
     t_end = Ribasim.seconds_since(cfg.endtime, cfg.starttime)
 
-    # demand[user_idx][priority](t)
+    # demand[user_idx][priority_idx](t)
```

Comment:
Why do we need this commented-out code?

Reply:
We don't, it was there as a reminder to myself.

Reply:
This wasn't really commented-out code though, it was documentation explaining what the two indices represented for the lines below:

```
    @test demand[1][2](0.5 * t_end) ≈ 1.0
    @test demand[2][1](0.0) ≈ 0.0
    @test demand[2][1](t_end) ≈ 0.0
```

Which I think is useful. But no need to put it back though if you don't need it.